stats/opencensus/opencensus.go

```diff
@@ -159,15 +159,16 @@ func getRPCInfo(ctx context.Context) *rpcInfo {
 	return ri
 }
 
-// GetTraceAndSpanID returns the trace and span ID of the span in the context.
-// Returns true if IDs present and false if IDs not present.
-func GetTraceAndSpanID(ctx context.Context) (trace.TraceID, trace.SpanID, bool) {
+// GetTraceAndSpanIDAndIsSampled returns the trace and span ID of the span in
+// the context, and if the span is sampled. Returns true (as the last bool) if
+// IDs present and false if IDs not present.
+func GetTraceAndSpanIDAndIsSampled(ctx context.Context) (trace.TraceID, trace.SpanID, bool, bool) {
```

Comment:
Please name these return values.

Reply:
Actually...can you put this data in a struct and return it + a bool instead?

And maybe name the function `GetTraceInfo(context.Context) (*TraceInfo, bool)`?

Reply:
Done. Switched to SpanInfoFromContext(ctx context.Context) (*SpanInfo, bool)